test/CMakeLists.txt

```diff
@@ -46,11 +47,17 @@ foreach (algo_index RANGE 29)# 43
     if (algo_index STREQUAL 10)
       set (enable_ FALSE)
     endif ()
+    # Check if LUKSAN targets are available.
     if (enable_)
       add_test (NAME testopt_algo${algo_index}_obj${obj_index} COMMAND testopt -r 0 -a ${algo_index} -o ${obj_index})
       if (CMAKE_HOST_SYSTEM_NAME MATCHES Windows)
         set_tests_properties (testopt_algo${algo_index}_obj${obj_index} PROPERTIES ENVIRONMENT "PATH=${PROJECT_BINARY_DIR}\\${CMAKE_BUILD_TYPE};$ENV{PATH}")  # to load dll
       endif ()
+      if(NOT ${NLOPT_WITH_LUKSAN})
```

Comment:
you can just write `if(NOT NLOPT_WITH_LUKSAN)`
also the comment at line 50 and the index list can go there